crates/fuel-core/src/database.rs

```diff
@@ -102,6 +102,13 @@ impl Database<OnChain> {
             .map(|(_, block)| block)
             .ok_or_else(|| not_found!("FuelBlocks"))
     }
+
+    pub fn first_block_height(&self) -> StorageResult<BlockHeight> {
```

Comment:
You can re-use `genesis_block` function

<img width="779" alt="image" src="https://github.com/FuelLabs/fuel-core/assets/18346821/29be9769-83fc-47a7-966b-74f5e64703cc">

Reply:
Where's that defined? `rg 'fn genesis_block'` doesn't find it in this branch or `master`.

Reply:
Oh, my bad, it is not merged=D https://github.com/FuelLabs/fuel-core/pull/1826

Reply:
I'm going to keep it as-is now, and after that's merged we can combine the implementations.